src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4953,6 +5170,84 @@ namespace HVACVariableRefrigerantFlow {
             VRFTU(VRFTUNum).AvailStatus = ZoneComp(VRFTerminalUnit_Num).ZoneCompAvailMgrs(VRFTUNum).AvailStatus;
         }
 
+        if (MySuppCoilPlantScanFlag(VRFTUNum) && allocated(PlantLoop)) {
+            if ((VRFTU(VRFTUNum).SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) ||
+                (VRFTU(VRFTUNum).SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingSteam)) {
+
+                if (VRFTU(VRFTUNum).SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
```

Comment:
This nested IF structure is kinda hard to read, and it looks like all but the last line or two are in each of the nested sections.  I'd suggest just eliminating the first `if (Water Or Steam)` block as shown here:

```
if water or steam
  if water
    water-stuff
  else if steam
    steam-stuff
  end
  a couple common lines
end
```

to

```
if water
  water-stuff
  a couple common lines
else if steam
  steam-stuff
  a couple common lines
end
```

I realize this code isn't called frequently, so the performance impact of the nested structure will be minimal, and it's not a functional change, so if no other changes are needed, I'll not worry about it.

Reply:
@Myoldmopar I am making these changes.